<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geomdef</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Imege"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRotate"/>
    <addaction name="actionRotate_1"/>
    <addaction name="actionSkew"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGeomdef_Help"/>
    <addaction name="actionWeb_Site"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/fileopen.png</normaloff>:/image/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/zoomout.png</normaloff>:/image/icons/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/zoomin.png</normaloff>:/image/icons/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/pointrot.png</normaloff>:/image/icons/pointrot.png</iconset>
   </property>
   <property name="text">
    <string>Rotate around piont</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRotate_1">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/axisrot.png</normaloff>:/image/icons/axisrot.png</iconset>
   </property>
   <property name="text">
    <string>Rotate around axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSkew">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/skew.png</normaloff>:/image/icons/skew.png</iconset>
   </property>
   <property name="text">
    <string>Skew</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionBeczka_poduszka">
   <property name="text">
    <string>beczka/poduszka</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/filesave.png</normaloff>:/image/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/filesaveas.png</normaloff>:/image/icons/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/fileprint.png</normaloff>:/image/icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPrinter_Setup">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Printer Setup</string>
   </property>
  </action>
  <action name="actionClose_Imege">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/editdelete.png</normaloff>:/image/icons/editdelete.png</iconset>
   </property>
   <property name="text">
    <string>Close Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/exit.png</normaloff>:/image/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGeomdef_Help">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/help.png</normaloff>:/image/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Geomdef Help</string>
   </property>
  </action>
  <action name="actionWeb_Site">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/gohome.png</normaloff>:/image/icons/gohome.png</iconset>
   </property>
   <property name="text">
    <string>Web Site</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="reso.qrc">
     <normaloff>:/image/icons/mail_generic.png</normaloff>:/image/icons/mail_generic.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </action>
  <action name="actionE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="reso.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_In</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_Out</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRotate</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRotate_1</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>rotateoDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSkew</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>skewd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFileAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrint</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose_Imege</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGeomdef_Help</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWeb_Site</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openWeb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFile()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>rotateDialog()</slot>
  <slot>rotateoDialog()</slot>
  <slot>skewd()</slot>
  <slot>saveFile()</slot>
  <slot>saveFileAs()</slot>
  <slot>clearImage()</slot>
  <slot>print()</slot>
  <slot>exit()</slot>
  <slot>activateMenu()</slot>
  <slot>openHelp()</slot>
  <slot>aboutInfo()</slot>
  <slot>openWeb()</slot>
 </slots>
</ui>
